### DISCUSSION TITLE
pytube error

### DISCUSSION THREAD

**Discussion by SLV54** [n3d1117/chatgpt-telegram-bot#383] [Q&A]
Hi All.
I'm trying plugins. Weather, voice acting, - Ok.
When extracting audio from youtube video i get this error:
`root - WARNING - Failed to extract audio from YouTube video: get_transform_object: could not find match for var for={(.*?)};`

The request in the SSH console: `pytube <URL> -a` is working fine.
What can be fixed

**Comment by n3d1117** [comment]
I have no idea, you could try opening an issue in the [pytube repo](https://github.com/pytube/pytube)
